__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const data = require("../db/data/test-data")
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection")
+const app = require("../app")
+const request = require("supertest")
+
+beforeAll(() => seed(data))
+afterAll(() => db.end())
+
+describe("GET/topics", () => {
+    test("200: Should respond with all topics in an array of objects containing the properties slug and description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const topics = body;
```

Comment:
Rather than sending your data directly back on the key of topics, send your data back on the key describing what the data is i.e.  our response object should look like:
`response: { body : { topics : <topics data arr here> } }`

instead of 
`response: { body : <topics data arr here> }`